### ISSUE TITLE
Strange behaviour when instantiated without new.

### ISSUE THREAD

**Issue by brugnara** [victorquinn/chancejs#157]
I've tried:

```js
var chance = require('chance')();
console.log(chance.hash({length:15}));
```

But this outputs always the **same** hash **each time** I relaunch the app.

With a **new** it works as expected.

```js
var chance = new (require('chance'))();
```

**Comment by oliversalzburg** [comment]
As per the documentation, `require('chance')` returns the constructor. The suggest usage is:

```js
var Chance = require('chance'),
    chance = new Chance();
```

**Comment by brugnara** [comment]
Yes, but the code has a `factory` returning a `new Chance()` and this is not working property.

**Comment by vvo** [comment]
I confirm this using browserify, chance.Chance() will always generate the same string(), which is very funny

**Comment by victorquinn** [comment]
I suspect this was fixed with #153 I just haven't cut a release yet with that change.

I'm going to add a unit test and, if it works, deploy version 0.7.4 with that pulled in.
